components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -88,6 +89,11 @@
 
   // -- Constants --
 
+  private static final String ALLOW_AUTOSTITCHING_KEY = "zeissczi.allow.autostitch";
```

Comment:
As discussed in the meeting, I would err on 1- not using classname, 2- using lowercase, 3- keeping names short while descriptive as much as possible.

In the case above, my preference goes with `zeissczi.autostitch` for the first option, for the second option, I don't have any good alternative to `zeissczi.include.attachments` to propose.